projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/babel.config.js

```diff
@@ -1 +1,6 @@
+/*
```

Comment:
To add to what @sabadzhiev said in another comment, I'm not sure if we can copyright a configuration. In this case, it's an Apache license, so it might not matter.